### DISCUSSION TITLE
Enable run locally and import csv in under a minute

### DISCUSSION THREAD

**Discussion by mdav43** [cosmograph-org/cosmos#61] [Ideas]
Hi, are generated an actual issue ticket, but wanted to put it in the discussion forum as well. 

It would be great to have a repo that we could clone and run locally that Mira is the functionality on the main website. For example, the ability to import a CSV and have this up and running in under a minute.

**Comment by rokotyan** [comment]
Hi @mdav43,

The code of Cosmograph.app is not open source and we're not planning to do it, at least for now.

Feel free to send us an email to hi@cosmograph.app if you want to discuss potential licensing.
